### DISCUSSION TITLE
[DONE ✅] [UI/UX] Active state to header navigation

### DISCUSSION THREAD

**Discussion by dstrukt** [btcpayserver/btcpayserver#2364] [General]
Currently, we only have visible hover states, but not active states for the page/subpage you're currently on.

I would like to propose that add a simple active state to those pages, and remove the hover white bar.

Before:
<img width="1360" alt="Screen Shot 2021-03-14 at 2 27 37 PM" src="https://user-images.githubusercontent.com/6250771/111084819-8fc4c200-84d1-11eb-8c56-058268389736.png">
(hover)
<img width="1302" alt="Screen Shot 2021-03-14 at 2 27 44 PM" src="https://user-images.githubusercontent.com/6250771/111084821-905d5880-84d1-11eb-9b57-25dc0678dde8.png">

After:

<img width="1696" alt="Screen Shot 2021-03-14 at 3 19 04 PM" src="https://user-images.githubusercontent.com/6250771/111086320-b2a6a480-84d8-11eb-9118-d0c017ba27dc.png">

Still working on colors (will update this), but wanted to get this discussion started.

**Comment by pavlenex** [comment]
Personally, I like the iteration on the left. Should we consider people who may be visually impaired when we're designing links, hoovers, links, etc?

**Reply by Zaxounette** [reply]
For people with vision deficiencies, there could be added two different options to cater to the most frequent pathologies.

First, a `blind mode` that would consist of enlarged typos designed for users with near blind vision or extremely low vision.
Second, a `color blind mode`, consisting of color changes in the UI for better visualization designed for people with near proper vision but are color blinded.
Both options together can cater to the users with both pathologies at a time.

**Reply by pavlenex** [reply]
I don't think developing a separate mode is cost-effective, there is already a [set](https://www.interaction-design.org/literature/topics/accessibility) of accessibility [standards](https://medium.com/salesforce-ux/7-things-every-designer-needs-to-know-about-accessibility-64f105f0881b) that makes sure visually impaired people can use the UI. So IMO, in the future, we probably want to design the UI with accessibility in mind. This is probably a good topic for a separate discussion.

**Reply by dstrukt** [reply]
Yeah, I've actually been working through that (contrast, etc..) which is why getting the colors to @dennisreimann has been taking longer than anticipated.

Completely agreed that should be applied here though, I just wanted to get the ball rolling, as swapping the colors is arguably the easiest part of the dev work.

Also agreed on the left, just wanted to show another iteration incase people were curious.

**Reply by bolatovumar** [reply]
Iteration on the left is definitely better in terms of a11y as it has way more contrast.
